torch/autograd/function.py

```diff
@@ -372,3 +381,71 @@ def forward_extended(self, *input):
 
     def backward_extended(self, *grad_output):
         raise NotImplementedError
+
+
+def symbolic_override(symbolic_fn):
+    """
+    Decorator to override ONNX export of the a given Python function with
+    specified subgraph.
+
+    Effectively allows to attach symbolic() implementation to an arbitrary
+    python function. Requirements to the decorated function:
+     - being non-member function
+     - positional inputs are Variables/Tensors or (nested) lists or tuples of
+       them (similar requirement to NestedIOFunction)
+     - outputs are similarly Variables/Tensors or (nested) lists or tuples of
+       them
+     - keyword arguments are of non-tensor type
+     - note, that function does NOT have to be autograd.Function
```

Comment:
You said that it should be a regular Python function, so it's not even obvious that it can be an autograd Function